p2p/discovery.go

```diff
@@ -99,17 +97,14 @@ func (d *discovery) startMdns() {
 // HandlePeerFound is event handler called when a peer is found with discovery
 func (n Notifee) HandlePeerFound(p peer.AddrInfo) {
 	log.Trace(
-		"Peer found using mDNS discovery service",
-		"host", n.host.ID(),
+		"Peer found using discovery",
```

Comment:
being nitpicky but I feel like the "using discovery" is a bit redundant, are there other ways we would find peers?

Reply:
You're right, not sure how else we would "find peers".

Reply:
fixed